timesketch/lib/analyzers/interface.py

```diff
@@ -1150,3 +1165,196 @@ def get_kwargs(cls):
     def run(self):
         """Entry point for the analyzer."""
         raise NotImplementedError
+
+
+class AnalyzerOutputException(Exception):
+    """Analyzer output exception."""
+
+
+class AnalyzerOutput:
+    """A class to record timesketch analyzer output.
+
+    Attributes:
+        platform (str): [Required] Analyzer platfrom.
+        analyzer_identifier (str): [Required] Unique analyzer identifier.
```

Comment:
The analyzer_name is uniq as well. What is the identifier here?

Reply:
The analyzer_name is the display name and the analyzer_identifier is the unique analyzer name. The reason we did not go with "analyzer_name" as the unique identifier, is the potential for reuse of the output schema with other tools like turbinia analyzers.